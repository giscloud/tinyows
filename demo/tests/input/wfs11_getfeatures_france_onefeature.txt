service=WFS&version=1.1.0&request=GetFeature&typename=tows:france&maxfeatures=1
