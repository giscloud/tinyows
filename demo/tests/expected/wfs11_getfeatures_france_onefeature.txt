Content-Type: text/xml; subtype=gml/3.1.1

<?xml version='1.0' encoding='UTF-8'?>
<wfs:FeatureCollection
 xmlns:tows='http://www.tinyows.org/'
 xmlns:wfs='http://www.opengis.net/wfs'
 xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
 xmlns:gml='http://www.opengis.net/gml'
 xmlns:xsd='http://www.w3.org/2001/XMLSchema'
 xmlns:ogc='http://www.opengis.net/ogc'
 xmlns:xlink='http://www.w3.org/1999/xlink'
 xmlns:ows='http://www.opengis.net/ows'
 xsi:schemaLocation='http://www.tinyows.org/
    http://127.0.0.1/cgi-bin/tinyows?service=WFS&amp;version=1.1.0&amp;request=DescribeFeatureType&amp;Typename=tows:france   http://www.opengis.net/wfs
   http://schemas.opengis.net/wfs/1.1.0/wfs.xsd
   http://www.opengis.net/gml
   http://schemas.opengis.net/gml/3.1.1/base/gml.xsd'
>
  <gml:featureMember>
   <tows:france gml:id="france.1">
   <tows:id_geofla>49</tows:id_geofla>
   <tows:code_chf_l>053</tows:code_chf_l>
   <tows:nom_chf_l>BOURG-EN-BRESSE</tows:nom_chf_l>
   <tows:x_chf_lieu>8231</tows:x_chf_lieu>
   <tows:y_chf_lieu>21379</tows:y_chf_lieu>
   <tows:x_centroid>8244</tows:x_centroid>
   <tows:y_centroid>21380</tows:y_centroid>
   <tows:nom_dept>AIN</tows:nom_dept>
   <tows:code_reg>82</tows:code_reg>
   <tows:nom_region>RHONE-ALPES</tows:nom_region>
   <tows:code_dept>01</tows:code_dept>
   <tows:geom><gml:MultiSurface srsName="urn:ogc:def:crs:EPSG::27582"><gml:surfaceMember><gml:Polygon><gml:exterior><gml:LinearRing><gml:posList>801150 2092615 800669 2093190 800688 2095430 800780 2095795 800589 2096112 800333 2097190 799095 2098505 799136 2098838 799690 2100360 799329 2100395 797715 2100950 797655 2101330 797410 2102850 797149 2102788 796755 2102425 796738 2102785 796612 2103487 795234 2103892 794905 2104515 794547 2104359 791555 2102570 791465 2102760 791935 2104075 791835 2105023 791670 2105295 790205 2105555 790035 2105875 789304 2107116 788617 2107148 788005 2106760 787670 2106708 786705 2106990 786350 2107029 785720 2107335 785562 2108399 786670 2109486 786930 2109775 787055 2110010 787208 2110344 787367 2111056 786900 2113550 786795 2114260 786830 2114370 786811 2114751 786384 2116197 786550 2116940 786560 2117563 786465 2117860 785956 2119206 787211 2120474 787415 2120775 787600 2121475 787536 2121809 787137 2122743 786610 2123110 786457 2124100 787138 2124890 787365 2125160 787545 2125377 787880 2125830 788198 2126745 788225 2127075 789130 2128290 789370 2128585 789790 2129209 789995 2129525 790445 2131460 790237 2131793 789585 2132230 788993 2132742 788830 2133100 788650 2133655 788638 2134028 788673 2135142 789501 2136382 789510 2137110 789596 2138174 789680 2138520 789845 2139015 790050 2139590 790594 2140547 790750 2142011 790740 2142380 790739 2142716 791139 2143599 791749 2144456 791860 2144790 792199 2147049 792285 2147420 792880 2148815 793615 2150425 793737 2150782 793650 2152285 793618 2152660 793480 2153400 794012 2155103 794347 2155696 794535 2155980 794686 2156241 795135 2157025 796115 2159030 796193 2160314 796250 2160635 796311 2161408 796335 2161795 796324 2162102 796316 2163021 796385 2163470 796495 2163785 797830 2165380 797970 2165700 797959 2167362 797910 2167695 797905 2167835 798850 2169845 798930 2170117 798980 2170395 799122 2171035 799255 2171335 799340 2171435 800295 2171593 800326 2170924 800757 2170498 801505 2170754 802043 2171611 802820 2171760 804846 2171217 805165 2171380 805338 2170485 805485 2170210 808352 2168729 808655 2168520 809605 2168719 809930 2168735 811710 2170150 813076 2169640 814292 2170568 814670 2170640 815189 2171442 816978 2171075 817045 2172530 818290 2171719 819798 2171534 819990 2171205 819753 2170948 820496 2169812 820305 2169145 820810 2168648 821023 2167568 821020 2167190 821753 2167185 822453 2166355 822690 2166075 823263 2166394 823585 2166285 824250 2165476 825284 2165330 825600 2165160 828440 2165065 829372 2163119 828470 2162490 828705 2161655 828395 2161531 827745 2161519 828485 2161005 830445 2159908 831525 2159915 832747 2158958 833030 2158780 833286 2158494 833885 2158105 833687 2157807 832940 2157040 832830 2156696 833866 2155806 833825 2155445 833752 2155097 833675 2154750 835995 2153047 835930 2152675 836194 2151171 836185 2150780 836190 2150680 836405 2150149 836690 2150070 837059 2153840 837545 2154400 838628 2151187 838735 2150820 840056 2151192 840530 2151708 840855 2151840 841371 2151081 841670 2150965 841591 2150606 841093 2148479 840525 2148155 840483 2146631 841576 2145603 841898 2145394 843750 2145800 843973 2146033 844372 2145523 844675 2145410 846805 2146002 848292 2147688 848440 2148035 848651 2148733 850029 2148841 850390 2148775 851184 2149407 851650 2150710 851857 2150922 851710 2151180 852195 2152638 853883 2153465 854050 2153815 854684 2154550 854950 2153948 855010 2153605 854706 2152546 856495 2152522 857425 2151954 857675 2151680 857795 2151135 860115 2150917 860522 2149401 860356 2147845 860868 2145955 861160 2145690 861499 2145842 861800 2146060 863931 2146529 864270 2146675 867954 2146311 868320 2146245 868625 2146162 868940 2146145 870819 2146227 871569 2147047 872099 2147009 872230 2146655 873891 2148801 874080 2149145 875215 2148900 875688 2149908 875836 2151749 876350 2152270 877249 2151866 877575 2151825 877751 2152160 879950 2156500 880577 2157904 882757 2159490 884049 2160898 885600 2163565 886165 2164270 886435 2164140 889069 2163415 889778 2162493 890867 2162099 893000 2160395 894101 2160083 894575 2159465 894622 2158743 893375 2156340 892712 2155944 892540 2155185 891600 2153375 891281 2152677 891245 2152295 891340 2150917 890292 2149984 890080 2149695 890987 2147869 891235 2147615 891560 2147405 891794 2146272 891337 2145258 890413 2144570 889185 2145445 888822 2145356 887853 2144792 887180 2145110 886092 2143618 885735 2143730 885432 2143963 884760 2143997 883760 2143495 883455 2143280 882749 2142419 882058 2142149 881989 2141782 880591 2141448 880422 2140052 879890 2139555 879855 2139435 882197 2138192 882019 2137078 881408 2136753 880810 2134940 880202 2133632 880177 2132631 879600 2132205 879351 2132001 878142 2131755 877449 2131650 876853 2131962 875845 2130970 875089 2130628 874800 2130520 874322 2129464 874925 2128114 874725 2126982 872900 2126720 872100 2126810 871837 2127080 870648 2128513 869972 2128829 869296 2128495 869435 2127050 868710 2125650 868663 2125275 868525 2123780 868760 2123569 868985 2123070 868525 2121885 868617 2121591 868760 2120685 868864 2120349 869034 2119679 868545 2118760 868661 2118458 868975 2117935 868911 2117610 869020 2116295 869215 2115554 870970 2114005 870979 2113621 870925 2112855 871130 2111817 870721 2110485 870455 2110250 870800 2110165 870344 2109573 870311 2108843 870897 2107500 870710 2106390 870615 2106018 869975 2103410 869465 2101545 869314 2101232 869040 2099875 868989 2099512 868710 2098080 868050 2097320 867940 2096964 867995 2094425 867975 2094056 867825 2092595 867736 2092277 867995 2091670 867979 2091277 867788 2090131 868170 2088645 867935 2088380 867643 2088330 867110 2088110 867246 2087753 867588 2086334 866723 2085607 866013 2084278 865294 2084187 864275 2084575 864000 2084465 863659 2084328 861885 2084450 861845 2084109 862185 2083150 862398 2082820 862694 2082092 862533 2081333 860925 2079632 860842 2078870 861160 2078155 861300 2077808 861210 2077080 860973 2076787 860337 2076386 858511 2076155 858205 2075110 858072 2074811 857453 2074591 857320 2073960 857265 2073584 856650 2073221 856355 2073460 856182 2073797 855610 2074780 854744 2075941 854474 2077758 852864 2078730 852620 2079020 852647 2079386 851685 2079920 851354 2080021 850753 2079717 850470 2079917 850320 2080840 850069 2081131 850023 2082613 850741 2082507 851509 2081230 852157 2081554 852073 2082291 850451 2083206 850375 2083940 850111 2084174 849805 2084350 848375 2084500 848150 2084810 847545 2086215 847281 2086449 845980 2087620 845694 2087856 844840 2088564 842895 2091270 842098 2091898 841825 2092105 841604 2092424 841170 2093065 840895 2093318 839825 2094355 839769 2096198 840798 2096536 840745 2096910 840580 2097185 840305 2097470 839320 2098117 839165 2098880 839016 2099191 838200 2099825 836342 2100899 836100 2101165 835840 2101453 834252 2102554 833175 2102260 833017 2102031 832785 2101525 832719 2101162 832433 2100482 831189 2099710 830330 2098550 829565 2095140 829550 2094745 829422 2094389 828638 2093577 827810 2091920 827523 2091724 826955 2091325 826653 2091135 825785 2090510 825487 2090257 824587 2089517 822700 2089735 822339 2089692 821748 2090044 821660 2090765 821459 2091109 820815 2092110 820470 2092273 819281 2093148 818215 2093260 817851 2093406 816405 2093980 815735 2093715 815844 2092629 815501 2092497 814830 2094120 814502 2094025 814175 2093925 813785 2093907 811510 2093620 811202 2093793 810503 2093796 809935 2093390 808612 2093210 808305 2093030 807965 2092976 807085 2093490 806759 2093370 806450 2093210 805480 2092960 804700 2093105 804426 2093280 803230 2093210 802864 2093269 801150 2092615</gml:posList></gml:LinearRing></gml:exterior></gml:Polygon></gml:surfaceMember></gml:MultiSurface></tows:geom>
   </tows:france>
  </gml:featureMember>
</wfs:FeatureCollection>
